### ISSUE TITLE
Choisir la longeur de l'échantillon dans l'IGU

### ISSUE THREAD

**Issue by trebmuh** [linuxmao-org/Frontieres#15]
Actuellement, les échantillons affichés avec lesquels on peut jouer ne sont qu'une partie des échantillons chargés. il suffit pour s'en convaincre d'écouter les échantillons en dehors du logiciel.

Ça serait intéressant de pouvoir choisir la taille/longueur de l'échantillon à afficher dans le logiciel grâce à un [clic-droit](https://github.com/linuxmao-org/Borderlands/issues/11) par exemple.

**Comment by jpcima** [comment]
Tu veux dire, pouvoir réduire l'espace occupé par un échantillon sur l'affichage ?

Si je peux ajouter un truc, il m'est venu à l'idée que ce serait bien de pouvoir balader la vue au moyen d'un clic maintenu du bouton central de la souris. Il me semble que Blender fonctionne ainsi.
Cela permettrait d'accéder à un plus grand espace de travail.

**Comment by trebmuh** [comment]
> Tu veux dire, pouvoir réduire l'espace occupé par un échantillon sur l'affichage ?

Non, ça c'est déja pris en charge en cliquant dessus, puis sans lâcher le clic, en appuyant sur la touche R, puis sans lâcher tout ça, en déplaçant la souris.

Ce que je veux dire ici, c'est que les échantillons chargés ne le sont pas entièrement, ce qui est chargé, c'est juste quelque secondes de l'échantillon. Ce que j'imagine ici, c'est qu'on puisse choisir genre "charge moi de la 23ème seconde à la 41ème de l'échantillon truc.wav" par exemple.

**Comment by olof29** [comment]
ça reveindrait à pouvoir deplacer le début et la fin de la fenetre par rapport à l'echantillon, et non en redimentionnant l'echantillon ?

**Comment by trebmuh** [comment]
pas seulement, ça pourrait aurait faire en sorte qu'en temps qu'utilisateur, je puisse choisir de lire disons de la seconde "0" à la seconde "6", puis qu'après, je puisse choisir de changer ça pour la seconde "0" à la seconde "15", et encore après, de la seconde "7" à la seconde "52",...
